torch/quantization/observer.py

```diff
@@ -97,9 +105,43 @@ class MinMaxObserver(ObserverBase):
     __annotations__ = {'min_val' : Optional[torch.Tensor], 'max_val' : Optional[torch.Tensor]}
 
     def __init__(self, **kwargs):
+        #  For x86 quantized kernels, we need to ensure that the vpmaddubsw instruction
+        #  does not overflow. We allow for a reduce_range argument to observers that
+        #  reduces the quantized range to (0,127) or (-64, 63).
+        # . This is not the optimal choice for non x86 backends as
+        #  lose a bit of precision for activations.
+        #
+        #   FBGEMM uses vpmaddubsw instruction to multiply activations (uint8_t) and
+        #   weights (int8_t).
+        #
+        #  https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm256_maddubs_epi16&expand=3284,3530
```

Comment:
May be for this part ... we can just point to the comment in the qconv.cpp file rather than repeating it.

Reply:
Ok, will abbreviate it and point to the comment